packages/playground/glob-import/index.html

```diff
@@ -52,3 +53,22 @@
     2
   )
 </script>
+
+<script type="module">
+  // issue #7307
+  const relativeRawModules = import.meta.globEager(
+    '../dynamic-import/nested/shar*.js',
```

Comment:
I think it would be safer if we can glob a file within the `glob-import` playground. Otherwise a change in the `dynamic-import` playground could cause unexpected test failures for future contributors.

Reply:
I'm ok with limiting the test to `glob-import`, but it makes the test a tad contrite, since the purpose is to test directory tree traversal above the parent folder which in this case is `glob-import` itself. Navigating back into `glob-import` would be a use-case we wouldn't expect in real life. However, the test is probably robust enough, and at the very least a reminder to evaluate the possibility.